rpc/src/rpc.rs

```diff
@@ -3396,7 +3396,7 @@ pub mod rpc_full {
                 .unwrap_or(0);
 
             let durable_nonce_info = transaction
-                .get_durable_nonce()
+                .get_durable_nonce(true)
```

Comment:
Any reason not to query the feature from `preflight_bank` here?

Reply:
I think it's fine to hardcode here because it just means that RPC could treat a transaction as non-durable while the runtime still treats it as a durable tx before the feature switch is enabled. At worst, this just means that the transaction might have a `last_valid_block_height` of 0 (which means it would only be sent once, never retried). Not a big deal since the tx would fail anyways.

Reply:
I was thinking that we should reject all transactions that contain to nonce accounts in preflight.  I could use the preflight bank instead, maybe that's more clear and consistent

Reply:
I'm fine with either way, don't think this will cause any issues

Reply:
I was just thinking that the error behavior would be pretty confusing if preflight doesn't match submission

Reply:
Agree, I changed it